src/integration/testdata/simple.yml

```diff
@@ -22,6 +22,14 @@ exporters:
 
 service:
   pipelines:
+    logs:
+      receivers: [ otlp/test ]
+      processors: [ batch ]
+      exporters: [ otlp, file ]
+    traces:
+      receivers: [ otlp/test ]
+      processors: [ batch ]
+      exporters: [ otlp, file ]
```

Comment:
Can you please match the buffer space provided in the array with that in the metrics section?